packages/riipen-ui/src/components/AppBar.test.jsx

```diff
@@ -0,0 +1,92 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import AppBar from "./AppBar";
+
+describe("<AppBar>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<AppBar />);
+    } catch (theError) {
+      error = theError;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  it("sets correct default props", () => {
+    const defaultProps = new AppBar().type.defaultProps;
+
+    const wrapper = mount(<AppBar />);
+
+    const component = wrapper.find("AppBar").childAt(0);
+    expect(component.hasClass(defaultProps.color)).toEqual(true);
+    expect(component.hasClass(defaultProps.position)).toEqual(true);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("displays custom label from child", () => {
```

Comment:
This name is sort of confusing. Looking at the details for `<AppBar>` if you pass it `children` it simply renders them. There is no mention of a "label".

Would maybe rename this to "displays given children" or something.